spec/rubocop/cop/rspec/capybara/specific_matcher_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Capybara::SpecificMatcher do
+  it 'does not register an offense for abstract matcher when elements with' \
+    'no matcher specified in detail' do
+    expect_no_offenses(<<-RUBY)
+      expect(page).to have_selector('article')
+      expect(page).to have_none_of_selectors('body')
+      expect(page).to have_css('tbody')
+    RUBY
+  end
+
+  it 'does not register an offense for abstract matcher when elements with' \
+    'not element' do
+    expect_no_offenses(<<-RUBY)
+      expect(page).to have_no_css('.a')
+      expect(page).to has_css?('#button')
+      expect(page).to has_no_css?('[table]')
+    RUBY
+  end
+
+  it 'registers an offense when using `#have_selector`' do
+    expect_offense(<<-RUBY)
+    expect(page).to have_selector('button')
+                    ^^^^^^^^^^^^^^^^^^^^^^^ Use `#have_button` instead of `#have_selector`.
+    expect(page).to have_selector('a')
+                    ^^^^^^^^^^^^^^^^^^ Use `#have_link` instead of `#have_selector`.
```

Comment:
I couldn't quickly find any examples in our code where we prefix methods with a `#` in offence messages. Is it necessary?

Reply:
Oops, that's right.
When I ran the `new_cop` command, it generated one with a prefix of `#`, so I completely thought this was the standard.

```console
bundle exec rake 'new_cop[Foo/Bar]' 
```

```ruby
        MSG = 'Use `#good_method` instead of `#bad_method`.'
```